ui/app/templates/components/secret-edit-metadata.hbs

```diff
@@ -18,7 +18,7 @@
           @modelValidations={{this.modelValidations}}
           @mode={{@mode}}
         />
-        <label class={{if (eq @mode "create") "title has-padding-top is-5" "title has-padding-top is-4"}}>
+        <label class={{concat "title has-top-padding-m " (if (eq @mode "create") "is-5" "is-4")}}>
```

Comment:
I don't think you need to use `concat` here. You should be able to move the `if` condition inside the quotes:

```hbs
<label class="title has-top-padding-m is-{{if (eq @mode 'create') '5' '4'}}">
```

Reply:
Great catch, thank you.